rpc/kate-rpc/src/lib.rs

```diff
@@ -173,8 +197,13 @@ where
 				.block
 				.extrinsics()
 				.iter()
-				.filter_map(|opaque| UncheckedExtrinsic::try_from(opaque).ok())
```

Comment:
The problem arises here, filter_map shouln't be used here, there is no correct behavior that skips any of the extrinsics. In this instance (the runtime upgrade bug for increasing tx size), the UncheckedExtrinsic::try_from fails because the decoding happens in the native code, which has an old tx size (BoundedVec bound), while the submitted extrinsic with bigger size was submitted and was checked by runtime and accepted into block.